apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/context/TracingContext.java

```diff
@@ -460,6 +461,11 @@ private void finish() {
                 finishedSegment.setIgnore(true);
             }
         }
+
+        if (segment.createTime() < RemoteDownstreamConfig.Agent.INSTANCE_REGISTED_TIME) {
```

Comment:
You need to create a method for this. This is also required when inject/extract and segment send to oap-server.